package.json

```diff
@@ -510,14 +510,20 @@
     "watchify": "^3.7.0"
   },
   "files": [
-    "bin",
-    "images",
-    "lib",
+    "bin/{*mocha,options.js}",
+    "assets/growl/*.png",
+    "lib/**/*.{js,html}",
     "index.js",
```

Comment:
Does the increase in specificity buy us anything here? Obviously, `images` is wrong (why was directory renamed to begin with?) and should be corrected. Wonder why we didn't get bug reports on that...

Reply:
Myself included. Since Apple took away the ability to show a custom icon, I don't pay the Notification Center notices much mind anymore.

Reply:
@plroebuck The increased specificity avoids problems described in npm/npm-packlist#14.  I had published another package the other day which used `files` but wound up publishing a `.DS_Store` file... alas.

Reply:
@boneskull, that's [not supposed to be possible](https://docs.npmjs.com/files/package.json#files)...